micrometer-core/src/test/java/io/micrometer/core/instrument/binder/hystrix/MicrometerMetricsPublisherCommandTest.java

```diff
@@ -0,0 +1,152 @@
+        HystrixCircuitBreaker circuitBreaker = HystrixCircuitBreaker.Factory.getInstance(key, groupKey, properties, metrics);
+        SimpleMeterRegistry registry = new SimpleMeterRegistry();
+        MicrometerMetricsPublisherCommand metricsPublisherCommand = new MicrometerMetricsPublisherCommand(registry, key, groupKey, metrics, circuitBreaker, properties);
+        metricsPublisherCommand.initialize();
+
+        final int NUM_SECONDS = 5;
+
+        for (int i = 0; i < NUM_SECONDS; i++) {
+            new SuccessCommand(key).execute();
+            new SuccessCommand(key).execute();
+            new SuccessCommand(key).execute();
+            Thread.sleep(10);
+            new TimeoutCommand(key).execute();
+            new SuccessCommand(key).execute();
+            new FailureCommand(key).execute();
+            new FailureCommand(key).execute();
+            new SuccessCommand(key).execute();
+            new SuccessCommand(key).execute();
+            new SuccessCommand(key).execute();
+            Thread.sleep(10);
+            new SuccessCommand(key).execute();
+        }
+
+        Thread.sleep(500);
+
+        List<Tag> tags = Tags.zip("group", "MicrometerGROUP", "key", "MicrometerCOMMAND-A");
+        assertThat(registry.find("hystrix.execution").tags(Tags.concat(tags, "event", "success")).firstValue()).hasValue(40.0);
+        assertThat(registry.find("hystrix.execution").tags(Tags.concat(tags, "event", "timeout")).firstValue()).hasValue(5.0);
+        assertThat(registry.find("hystrix.execution").tags(Tags.concat(tags, "event", "failure")).firstValue()).hasValue(10.0);
+        assertThat(registry.find("hystrix.execution").tags(Tags.concat(tags, "event", "short_circuited")).firstValue()).hasValue(0.0);
+        assertThat(registry.find("hystrix.circuit.breaker.open").tags(tags).firstValue()).hasValue(0.0);
+    }
+
+    @Test
+    public void testOpenCircuit() throws Exception {
+        HystrixCommandKey key = HystrixCommandKey.Factory.asKey("MicrometerCOMMAND-B");
+        HystrixCommandProperties properties = new HystrixPropertiesCommandDefault(key, propertiesSetter.withCircuitBreakerForceOpen(true));
+        HystrixCommandMetrics metrics = HystrixCommandMetrics.getInstance(key, groupKey, properties);
+        HystrixCircuitBreaker circuitBreaker = HystrixCircuitBreaker.Factory.getInstance(key, groupKey, properties, metrics);
+        SimpleMeterRegistry registry = new SimpleMeterRegistry();
+        MicrometerMetricsPublisherCommand metricsPublisherCommand = new MicrometerMetricsPublisherCommand(registry, key, groupKey, metrics, circuitBreaker, properties);
+        metricsPublisherCommand.initialize();
+
+        new SuccessCommand(key).execute();
+        new SuccessCommand(key).execute();
+        new TimeoutCommand(key).execute();
+        new FailureCommand(key).execute();
+        new FailureCommand(key).execute();
+        new SuccessCommand(key).execute();
+
+        Thread.sleep(1000);
```

Comment:
I detest the `Thread.sleep` in a test. I'm happy to mark these tests as 'slow' or even 'disabled' to keep the build fast